target/ppc/mmu_common.c

```diff
@@ -386,9 +386,9 @@ static int get_segment_6xx_tlb(CPUPPCState *env, mmu_ctx_t *ctx,
                   "Check segment v=" TARGET_FMT_lx " %d " TARGET_FMT_lx
                   " nip=" TARGET_FMT_lx " lr=" TARGET_FMT_lx
                   " ir=%d dr=%d pr=%d %d t=%d\n",
-                  eaddr, (int)(eaddr >> 28), sr, env->nip, env->lr, (int)msr_ir,
-                  (int)msr_dr, pr != 0 ? 1 : 0,
-                  access_type == MMU_DATA_STORE, type);
+                  eaddr, (int)(eaddr >> 28), sr, env->nip, env->lr,
+                  (int)(env->msr & M_MSR_IR), (int)(env->msr & M_MSR_DR),
```

Comment:
Here it also seems it'd change the behavior (although it looks like it'd just print `0x10` and `0x8` instead of 1 for IR and DR, respectively)